src/translations/fr.json

```diff
@@ -1,9 +1,11 @@
 {
     "General": "General",
 
-    "LimitReductions" : "Abattements de limites",
+    "LimitReductions" : "Abattements",
```

Comment:
Rename by `Reduction` ?

Reply:
same as the previous remark i think its better to keep it as it is